teamB/kyoungchanCho/app.js

```diff
@@ -36,6 +36,22 @@ app.get("/ping", (req, res) => {
     res.json({ message : "pong"})
 })
 
+app.post("/users", async (req, res, next) => {
+    const { name, email, profile_image, password } = req.body
```

Comment:
conding의 naming style에는 CamelCase 와 snake_case가 있습니다.

node.js는 camelCase를 사용하며 javascript의 또한 필수적인것은 아니지만 대체로 CamelCase를 사용하고 있습니다.

다른 코드들과 통일성을 위해 요청의 바디에 들어오는 정보들을 구조 분해 할당을 통해 변수로 넣어줄 때도 CamelCase를 사용하여 주세요!

```suggestion
    const { name, email, profileImage, password } = req.body
```